neurogym/envs/contextdecisionmaking.py

```diff
@@ -135,9 +114,11 @@ def _setup_impl_spaces(self):
         )
 
         # Action space: fixation + choices
+        action_space_name = {"fixation": 0, "choice": list(range(1, self.dim_ring + 1))}
+
         self.action_space = spaces.Discrete(
-            1 + self.impl_dim_ring,
-            name={"fixation": 0, "choice": list(range(1, self.impl_dim_ring + 1))},
+            n=1 + self.dim_ring,
+            name=action_space_name,
```

Comment:
given that `self.dim_ring + 1` (or reverse order) come back a few times, would it make sense to give it a separate variable to describes what that represents? Maybe `action_space_size` (if I interpret it correctly).

On the other hand, it is unclear to me why `action_space_name` needs to be defined just to be immediately loaded into `self.action_space` and then not used again

Reply:
> given that `self.dim_ring + 1` (or reverse order) come back a few times, would it make sense to give it a separate variable to describes what that represents? Maybe `action_space_size` (if I interpret it correctly).

I think it's okay to leave it as it is, otherwise it's less readable in the code. 

> On the other hand, it is unclear to me why `action_space_name` needs to be defined just to be immediately loaded into `self.action_space` and then not used again

Just for clarity, same for `obs_space_name`.